README.md

```diff
@@ -248,11 +248,12 @@ for `EXPLAIN`, that also provides performance tips (Commercial Software).
 ### Videos
 * [Citus Data Youtube channel](https://www.youtube.com/channel/UC8jpoK1BqQhDh6HDGFnM_DA/videos) - Citus related videos
 * [EnterpriseDB Youtube channel](https://www.youtube.com/channel/UCkIPoYyNr1OHgTo0KwE9HJw) -  EnterpriseDB related videos
-* [PGConf US Youtube channel](https://www.youtube.com/pgconfus/) - Conference videos
+* [Postgres Conference Youtube channel](https://www.youtube.com/channel/UCsJkVvxwoM7R9oRbzvUhbPQ/videos) - Conference videos
 * [Scaling Postgres](https://www.scalingpostgres.com/) - Postgres video blog series by Creston Jamison
 
 ### Community
 * [Mailing lists](https://www.postgresql.org/list/) - Official mailing lists for Postgres for support, outreach, and more. One of the primary channels of communication in the Postgres community. 
-* [Slack](https://postgres-slack.herokuapp.com/) - Slack channel for Postgres with close to 5000 users
-* [#postgresql on Freenode](ttps://webchat.freenode.net/#postgresql) - The most popular IRC channel about Postgres on Freenode with close to 1000 users 
-* [Reddit](https://www.reddit.com/r/PostgreSQL/) - A reddit community for PostgreSQL users with close to 10000 users
+* [Reddit](https://www.reddit.com/r/PostgreSQL/) - A reddit community for PostgreSQL users with over 12000 users
+* [Slack](https://postgres-slack.herokuapp.com/) - Slack channel for Postgres with over 7000 users
+* [#postgresql on Freenode](https://webchat.freenode.net/#postgresql) - The most popular IRC channel about Postgres on Freenode with over 1000 users 
+* [Telegram](https://web.telegram.org/#/im?p=@pg_sql) - A Telegram community for PostgreSQL users with over 500 users
```

Comment:
https://t.me/postgreschat has 750+, with significant number of Postgres major contributors.

Reply:
And if we start looking at Telegram, then maybe let's add non-English ones, which are big:

- http://t.me/pgsql -- Russian,  4200+ people
- http://t.me/postgresqlbr -- PostgreSQL BR, 2300+ people

Reply:
Wow, those look better than the one I was aware of!  🎉

I'll adapt now, thank you.